contracts/bridge/L1ECOBridge.sol

```diff
@@ -254,40 +257,33 @@ contract L1ECOBridge is IL1ECOBridge, CrossDomainEnabledUpgradeable {
         }
     }
 
+    /**
+     * @inheritdoc IL1ECOBridge
+     */
     function rebase(uint32 _l2Gas) external {
         inflationMultiplier = IECO(ecoAddress).getPastLinearInflation(
             block.number
         );
 
         bytes memory message = abi.encodeWithSelector(
-            L2ECOBridge.rebase.selector,
+            IL2ECOBridge.rebase.selector,
             inflationMultiplier
         );
 
         sendCrossDomainMessage(l2TokenBridge, _l2Gas, message);
     }
 
     /**
-     * @dev Adds ETH balance to the account. This is meant to allow for ETH
-     * to be migrated from an old gateway to a new gateway.
-     * NOTE: This is left for one upgrade only so we are able to receive the migrated ETH from the
-     * old contract
-     */
-    function donateETH() external payable {}
-
-    /**
-     * @dev Performs the logic for deposits by informing the L2 Deposited Token
-     * contract of the deposit and calling a handler to lock the L1 funds. (e.g. transferFrom)
+     * @dev Performs the logic for deposits by informing the L2 ECO token
+     * contract of the deposit and pulling in the L1 funds from the depositor
      *
-     * @param _l1Token Address of the L1 ERC20 we are depositing
-     * @param _l2Token Address of the L1 respective L2 ERC20
+     * @param _l1Token Address of the L1 ECO token contract
+     * @param _l2Token Address of the L2 ECO token contract
      * @param _from Account to pull the deposit from on L1
      * @param _to Account to give the deposit to on L2
-     * @param _amount Amount of the ERC20 to deposit.
+     * @param _amount Amount of ECO to pull and deposit.
```

Comment:
nitpick, i think the 'pull' makes this more confusing, should be kept as is or 'to be deposited'